pkg/config/config.go

```diff
@@ -37,6 +38,17 @@ func addTo(o *ctrl.Options, cfg *configapi.Configuration) {
 		o.Metrics.BindAddress = cfg.Metrics.BindAddress
 	}
 
+	internalCertManagementEnabled := ptr.Deref(cfg.InternalCertManagement.Enable, false)
+	if o.Metrics.CertDir == "" && (cfg.Metrics.CertDir != "" || !internalCertManagementEnabled) {
```

Comment:
Could we default to /tmp/k8s-metrics-server if internalCertManagement is false?

That way we can remove some of this setting and checking for nil?

Reply:
forget this. I thought I was looking elsewhere.

Reply:
btw, I had an incorrect reference there, changed to `/tmp/k8s-metrics-server/serving-certs`